packages/path_provider/path_provider/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-## NEXT
+## 2.1.4
 
+* Adds Downloads directory support on Android
```

Comment:
This doesn't add support, it just requires it. Many, many clients of the plugin already have support without this PR.

It would be much more accurate to fold a version of this into the next bullet point, so explain why the minimum version is changing and what the impact is (i.e., that it ensures Android support for `getDownloadsDirectory`).

Reply:
Thanks for feedback! I've combined these points, as you suggested.